contracts/packages/bindings/src/query.rs

```diff
@@ -0,0 +1,42 @@
+use cosmwasm_schema::cw_serde;
+use cosmwasm_std::{CustomQuery, Addr, PageRequest};
+use cosmwasm_std::{Binary};
+use crate::key::Key;
+
+#[cw_serde]
+pub enum WardenProtocolQuery {
+    Warden(WardenQuery),
+}
+
+impl CustomQuery for WardenProtocolQuery {}
+
+#[cw_serde]
+pub enum WardenQuery {
+    AllKeys { pagination: PageRequest, derive_wallets: Vec<WalletType> },
+}
+
+pub type WalletType = i32;
```

Comment:
I made some changes so we don't have "wallets" anymore. This particular case has been replaced by "addresses" (in short: `WalletType` became `AddressType`).

Please update all the definitions in this file to reflect the new protobufs.